yarn.lock

```diff
@@ -0,0 +1,3511 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I don't think we need to check in a yarn.lock for the root. We use the lock files for caching node_modules in the circle ci environment, and so far we've only had to do it for the individual packages (bits, charts, and dashboards).

It might be worthwhile to add this file's path to the .gitignore file to prevent accidental check-ins.

Reply:
@lukasholcik, thoughts?

Reply:
remove it, thanks